frontend/lib/pages/onboarding-step-4.tsx

```diff
@@ -15,20 +15,24 @@ import { Modal } from '../modal';
 
 const blankInitialState: OnboardingStep4Input = {
   phoneNumber: '',
-  canWeSms: false,
+  canWeSms: true,
```

Comment:
So we're making this optional, but we're also making it opt-*out*, because we really want to be able to follow-up via SMS.